Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -368,11 +368,48 @@ extension VariableDeclSyntax {
                 typeName = vtype
             }
 
+            let storageType: VariableModel.MockStorageType
+            switch v.accessorBlock?.accessors {
+            case .accessors(let accessorDecls):
+                if accessorDecls.contains(where: { $0.accessorSpecifier.tokenKind == .keyword(.set) }) {
+                    storageType = .stored(needsSetCount: true)
+                } else if let getterDecl = accessorDecls.first(where: { $0.accessorSpecifier.tokenKind == .keyword(.get) }) {
+                    if getterDecl.body == nil { // is protoccol
+                        var getterEffects = VariableModel.GetterEffects.empty
+                        if getterDecl.effectSpecifiers?.asyncSpecifier != nil {
+                            getterEffects.isAsync = true
+                        }
+                        if let `throws` = getterDecl.effectSpecifiers?.throwsClause {
+                            if let throwsType = `throws`.type {
+                                getterEffects.throwing = .typed(errorType: throwsType.trimmedDescription)
+                            } else {
+                                getterEffects.throwing = .any
+                            }
+                        }
```

Comment:
```suggestion
                        if let `throws` = getterDecl.effectSpecifiers?.throwsClause {
                            getterEffects.throwing = .init(`throws`)
                        }
```

this code might be duplicated.